src/main.rs

```diff
@@ -58,7 +58,11 @@ fn run_app() -> std::result::Result<(), Error> {
 
     // Update all downloaded languages.
     if args.update_langs {
-        update_langs();
+        assert!(
+            !args.wordlist_url.ends_with("/"),
+            "URL should end with branch name in github without trailing /"
+        );
```

Comment:
This is fine, but please use the proper error handling instead of just asserting. You can see an example a little below in the code or right here:

```rust
            let error = cmd.error(
                clap::ErrorKind::MissingRequiredArgument,
                format!(
                    "The {} argument was not provided.\n\n\tEither provide it as an argument or pass it in from standard input.",
                    "<SEARCH_TERM>".green()
                )
            );
            clap::Error::exit(&error);
```

Reply:
I should also note that the repository doesn't necessarily have to be on GitHub so the error message should be something like

> The provided wordlist URL must not end with a trailing '/'